handbook/communication/content_guidelines/voice_and_tone.md

```diff
@@ -0,0 +1,95 @@
+# Voice and tone
+
+One of the ways we create a consistent and familiar experience for our users is by speaking with a consistent voice throughout our product and content while being aware of our tone.
+
+## What’s the difference between voice and tone?
+
+What’s the difference between voice and tone? Think of it this way: our **voice** is a personification of who we are. We always have the same voice. But, our **tone** changes to match our context and situation, usually through word choice.
+
+## Voice guidelines
+
+Our brand has a distinct voice and character. This voice and character is the basis of our writing for all of our digital platforms.
+
+When we write in our shared voice, we are always:
+
+### (This section is todo)
+
+- (This section is todo)
+    
+## Using tone
+
+Our tone is more than just the words we choose: it’s the way that we choose to share our personality through our voice.
+
+We can express any given message in wildly different ways. To find the right tone for any given message, consider the four dimensions of tone:
+
+1. Funny vs. serious.
```

Comment:
It would be great to have some examples here as well

Reply:
I've added (in the latest commit) a section that better shows how voice and tone can be combined using these dimensions of tone. It's not a perfect example of how it works at **Sourcegraph**, but I think it'll be useful for helping readers understand how the concepts work together. Wdyt?